frontend/src/bundles/talent-onboarding/context/form-submit-provider.context.tsx

```diff
@@ -0,0 +1,45 @@
+import { createContext } from 'react';
+
+import { useContext, useState } from '~/bundles/common/hooks/hooks.js';
+
+type FormSubmitContextType = {
+    submitForm: (() => Promise<boolean>) | null;
+    setSubmitForm: React.Dispatch<
+        React.SetStateAction<(() => Promise<boolean>) | null>
+    >;
+};
+
+type ProviderProperties = {
+    children?: React.ReactNode;
+};
+
+const FormSubmitContext = createContext<FormSubmitContextType | null>(null);
+
+// Custom hook to use the context
```

Comment:
why do we need this hook?

Reply:
@nikita-remeslov 

1. The context keeps and provides the form submission function. So that each step can set its own form submission function. And then we can call it from step-content.tsx
2. _useFormSubmit()_ is a wrapper around useContext that ensures that the context is accessed in a component wrapped by the FormSubmitProvider. 
But it can be changed to just _useContext()_, but then if accidentally use this context, we may get null